tools/makemanifest.py

```diff
@@ -199,7 +199,7 @@ def mkdir(filename):
 def freeze_internal(kind, path, script, opt):
     path = convert_path(path)
     if not os.path.isdir(path):
-        raise FreezeError("freeze path must be a directory")
+        raise FreezeError("freeze path must be a directory: %s" % path)
```

Comment:
please use `.format(...)` to be consistent with other string-formatting in this file

Reply:
Updated as requested + email added to initial commit